src/Lean/Server/Utils.lean

```diff
@@ -105,6 +105,13 @@ def toFileUri (fname : System.FilePath) : Lsp.DocumentUri :=
 
 open Lsp
 
+partial def firstDiffPos (a b : String) : String.Pos :=
```

Comment:
`a b` -> `old new` ?

Reply:
It would also be good to note that this returns an offset into the UTF-8 encoding, e.g. by renaming to `firstDiffUtf8Pos`.

Reply:
Okay, I've renamed the function and added a docstring.  I thought tha `String.Pos` was always supposed to be UTF-8 though.

> a b -> old new ?

The function is completely symmetric in its arguments, so I'd rather not attach different meaning to them.